examples/node/cli.rs

```diff
@@ -851,7 +849,7 @@ fn list_offers(node: &LightningNode) -> Result<()> {
                 }
             }
         }
-        println!("      Status:              {:?}", offer.status);
+        println!("      Status:             {:?}", offer.status);
```

Comment:
How do these changes (remove 1 whitespace) affect the reversal of the LN payout fee?

Reply:
Sorry I missed that one line.